### ISSUE TITLE
Cannot read properties of undefined (reading 'aggregate')

### ISSUE THREAD

**Issue by NatoBoram** [zenstackhq/zenstack#1406]
**Description and expected behavior**

When using the trpc plugin, there's table files generated at `routers/<table_name>.ts`. Let's take a table `api_key_org`.

```ts
export default function createRouter<Config extends BaseConfig>(
	router: RouterFactory<Config>,
	procedure: ProcBuilder<Config>,
) {
	return router({
		aggregate: procedure
			.input($Schema.Api_key_orgInputSchema.aggregate)
			.query(({ ctx, input }) =>
				checkRead(db(ctx).api_key_org.aggregate(input as any)),
			),
```

This outputs an error:

> TypeError: Cannot read properties of undefined (reading 'aggregate')

If we log `$Schema`, we get this:

```ts
[Module: null prototype] {
  default: {
    Api_key_orgInputSchema: {
```

So `$Schema.Api_key_orgInputSchema.aggregate` is actually at `$Schema.default.Api_key_orgInputSchema.aggregate`.

**Screenshots**

**Environment (please complete the following information):**

-   ZenStack version: "2.0.2"
-   Prisma version: "5.13.0"
-   Database type: Postgresql

**Additional context**

**Comment by ymc9** [comment]
Hi @NatoBoram , thanks for reporting this. Could you let me know what framework you're using? Do you use vite for bundling?

**Comment by NatoBoram** [comment]
I'm not using a framework, this is just an API from raw TypeScript files compiled with `tsc`. I'm just using Zenstack, tRPC, Express and Prisma.

**Comment by ymc9** [comment]
> I'm not using a framework, this is just an API from raw TypeScript files compiled with `tsc`. I'm just using Zenstack, tRPC, Express and Prisma.

Got it. I believe it has something to do with the interop between ESM and CJS modules. Do you have a sharable project? @NatoBoram

**Comment by ymc9** [comment]
Fixed in 2.1.0
